src/js/model/primitives/marks/Mark.js

```diff
@@ -99,7 +103,7 @@ Mark.prototype.dataset = function(id) {
     return this;
   }
 
-  this.from = undefined;
+  delete this.from;
```

Comment:
How I read this (and @arvind can confirm) is that this code is meant to unset `this.from` if the dataset method is called with arguments that cause it to fall through all the `else if`'s... so we either need to keep the `= undefined`, which feels janky b/c we try to avoid using undefined as a value, even tho it does work in strict mode; or delete the property, which feels more explicit and accomplishes the same end. Initializing it to undefined to begin with still feels weird, b/c just omitting it entirely ought to accomplish the same goals; but I fixed this lint error about use of `undefined` then decided I didn't want to get too deep in the weeds by addressing the other as well. I would revert this if you don't feel it's appropriate, but I don't think I'd do anything further to this method b/c this patch was just for lint cleanup and isn't related to the store or signals.